controllers/util/k8sutil.go

```diff
@@ -6,13 +6,12 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
```

Comment:
Observed a couple of times, pls have your formatter (just assuming) not to adjust imports incorrectly.